<launch>
    <arg name="marker_to_root_cartesian" default="[-0.2, 0.0, 0.0, 0.0, 0.0, 0.0]"/>
    <arg name="marker_frame_name" default="ar_marker_0"/>
    <arg name="root_frame_name" default="ra_base_link"/>
    <arg name="camera_frame_name" default="camera_link"/>
    <arg name="window_width" default="2000"/>
    <arg name="filtering" default="true"/>
    <arg name="continuous" default="true"/>
    <arg name="output_static" default=""/>

    <node pkg="sr_camera_pose_estimation" type="camera_transform_publisher.py" name="camera_transform_publisher" output="screen">
        <rosparam param="marker_to_root_cartesian" subst_value="True">$(arg marker_to_root_cartesian)</rosparam>
        <param name="marker_frame_name" value="$(arg marker_frame_name)" />
        <param name="root_frame_name" value="$(arg root_frame_name)" />
        <param name="camera_frame_name" value="$(arg camera_frame_name)" />
        <param name="window_width" value="$(arg window_width)" />
        <param name="filtering" value="$(arg filtering)" />
        <param name="continuous" value="$(arg continuous)" />
        <param name="output_static" value="$(arg output_static)"/>
    </node>
</launch>
